jenkins/requirements.txt

```diff
@@ -1 +0,0 @@
-requests==2.5.3
```

Comment:
Another option is to have it in each requirements file not have a top level requirements file that pulls them all in. Instead we'd have the travis.yml file explicitly ``pip install`` each project separately. That kind of makes sense since they're separate projects anyway?

Preferences?